### PR TITLE
Mark songs changed based on song-artist instead of link

### PR DESCRIPTION
**Pull request by Brainicism**

### CODE REVIEW

File: src/seed/seed_db.ts [hunk 1]
```diff
@@ -739,7 +754,12 @@ async function reloadAutocompleteData(): Promise<void> {
             const currentDate = new Date();
             if (songsRemoved.length) {
                 logger.info(`${songsRemoved.length} songs removed.`);
-                const description = `**${songsRemoved.length} songs removed**:\n${songsRemoved.map((x) => `- ${x}`).join("\n")}`;
+                const description = `**${songsRemoved.length} songs removed**:\n${songsRemoved
+                    .map(
+                        (x) =>
+                            `- '${x.song_name_en}' - ${x.artist_name_en}  (${standardDateFormat(x.publishedon)}) | ${x.link}`,
```
Comment (taahamahdi) [inline src/seed/seed_db.ts:760]:
```suggestion
                            `- '${x.song_name_en}' - ${x.artist_name_en} (${standardDateFormat(x.publishedon)}) | ${x.link}`,
```

File: src/seed/seed_db.ts [hunk 2]
```diff
@@ -766,7 +786,12 @@ async function reloadAutocompleteData(): Promise<void> {
 
             if (songsAdded.length) {
                 logger.info(`${songsAdded.length} songs added.`);
-                const description = `**${songsAdded.length} songs added**:\n${songsAdded.map((x) => `- ${x}`).join("\n")}`;
+                const description = `**${songsAdded.length} songs added**:\n${songsAdded
+                    .map(
+                        (x) =>
+                            `- '${x.song_name_en}' - ${x.artist_name_en}  (${standardDateFormat(x.publishedon)}) | ${x.link}`,
```
Comment (taahamahdi) [inline src/seed/seed_db.ts:792]:
```suggestion
                            `- '${x.song_name_en}' - ${x.artist_name_en} (${standardDateFormat(x.publishedon)}) | ${x.link}`,
```

File: src/seed/seed_db.ts [hunk 3]
```diff
@@ -708,29 +715,37 @@ async function reloadAutocompleteData(): Promise<void> {
...
                 .execute();
 
+            const generateSongArtistIdentifier = (
+                x: AvailableSongDeltaData,
+            ): string => `${x.song_name_en},${x.artist_name_en}`;
+
             const availableSongsAfterSet = new Set(
-                availableSongsAfter.map((x) => x.link),
+                availableSongsAfter.map(
+                    (x) => `${generateSongArtistIdentifier(x)}`,
```
Comment (taahamahdi) [inline src/seed/seed_db.ts:724]: The string conversion here and below can be removed

Comment (Brainicism) [reply src/seed/seed_db.ts]: im assuming u mean the template string

**Review by taahamahdi — APPROVED** [review]

**Pull request merged**